Source/Scene/Vector3DTileMeshes.js

```diff
@@ -1,440 +0,0 @@
-define([
```

Comment:
I think I am missing something - should these files be deleted?  I thought they were used by the b3dm transcode for classification via b3dm.

Reply:
The transcode uses `Vector3DTilePrimitive` which takes buffers and re-batching information.

Reply:
I merged in the latest from `b3dm-class` so you can run the Sandcastle examples.